cross/openzwave/Makefile

```diff
@@ -0,0 +1,30 @@
+PKG_NAME = openzwave
+PKG_VERS = $(PKG_SVN_REV)
+PKG_EXT = tar.gz
+PKG_DOWNLOAD_METHOD = svn
+PKG_DOWNLOAD_VCS_CLEAN = nope
```

Comment:
Worth noting that with this approach the modification to spksrc.download.mk is no longer necessary. Should it be removed?